### ISSUE TITLE
Save button shortcut

### ISSUE THREAD

**Issue by vts3rss** [shuowu/yi-note#86]
Thanks for the great extension. Can you add a shortcut for the save button? More precisely, so that you can assign it yourself in the chrome extension settings. Thank you. Apologies for my English, I'm not a native English speaker

**Comment by jarodise** [comment]
yes, hotkeys for save instead of clicking would be way more efficient.

**Comment by Das2020** [comment]
Yes please
